tests/unit/tst-completion.test.ts

```diff
@@ -1030,7 +1033,8 @@ describe("Text Completion", () => {
       const generatedCompletionData = generateCompletionData(
         tstText,
         completionPosition,
-        triggerCharacter
+        triggerCharacter,
+        cppFlagForTstTest
```

Comment:
Now it is an interface, can't we omit this parameter (apart from when we're actually doing a C++ test)?

Reply:
Yeah don't know why I didn't delete it in those cases.. deleted in 805ebe742f3dfb6d82093c3d9606af7a90511b9f .